src/content/docs/hooks/hooks.routes.js

```diff
@@ -39,4 +39,9 @@ export default [
         path: '/useobject',
         description: 'Return a memoized object that only changes if one or more of its values have changed.',
     },
-];
\ No newline at end of file
+    {
```

Comment:
Do the items follow a specific order or newest last?

Reply:
Good question, I try to order them from most useful to least useful, but it's hard to determine that at this point. So where you placed it is good enough.